src/style.css

```
@@ -1 +1,83 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+.header {
+  max-width: 1200px;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  margin: 0 auto;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  padding-left: 50px;
+}
+
+.link-logo {
+  font-size: 0;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  position: relative;
+  right: 50px;
+}
+
+.nav-list {
+  display: flex;
+  padding-bottom: 3px;
+}
+
+.nav__link {
+  position: relative;
+  font-size: 12px;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  padding: 23px 0;
+}
+
+.nav-item {
+  list-style-type: none;
+  padding: 0 10px;
+}
+
+.nav-item:first-child {
+  padding-left: 0;
+}
+
+.nav-item:last-child {
+  padding-right: 0;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav-item .is-active::after {
+  position: absolute;
+  content: "";
+  top: 56px;
+  width: 36px;
```

```suggestion
  width: 100%;
```
Ширина псевдоелемента має відповідати ширині самого пункту меню, а не бути вказаною явно